eo-runtime/src/test/eo/org/eolang/malloc-tests.eo

```diff
@@ -102,22 +102,16 @@
     b.eq 20
 
 # Test.
-[] > malloc-is-strictly-sized-bool-error-overflow
-  try > @
-    malloc.for
-      false
-      m.put 86124867.88 > [m]
-    QQ.io.stdout e > [e]
+[] > throws-on-overflow-boolean-malloc
```

Comment:
@maxonfjvipon same here

Reply:
here an error is returned because we're trying to put 8 bytes to memory allocated only for 1 byte, [here](https://github.com/objectionary/eo/blob/c19fd16c8b61e44368af84996ae7ddf30292f534/eo-runtime/src/main/java/EOorg/EOeolang/Heaps.java#L135)